u8timeseries/models/fft.py

```diff
@@ -49,14 +59,22 @@ def _check_approximate_seasonality(series: TimeSeries, seasonality_period: int,
     return order <= max_seasonality_order
 
 
-def _find_relevant_timestamp_attributes(series: TimeSeries):
-    """
+def _find_relevant_timestamp_attributes(series: TimeSeries) -> set:
+    """ Finds pd.Timestamp attributes relevant for seasonality.
```

Comment:
Should it be in new line?

Reply:
I probably misunderstood, but my thinking was that this first line (before any line break) will be used as title in the docs. So I assume that's wrong?